doc for forwarAuthToken (#476)
SAP/cloud-sdk: docs/js/features/connectivity/destination.mdx

Note that the SAP Cloud SDK offers also a `mockTestDestination()` method, which reads in a `systems.json` and `credentials.json` to create destinations.
The advantage of files is that they can be excluded from the repository since they contain sensitive information.

:::tip AuthToken Forwarding

In most cases the destination service should be used in production to fetch destination including auth token.
In case of frequent service to service communication, the destination service would mean a big overhead.

> You can use an environment destination to overcome this issue.

Author: I tried to be more explicit. Have a second look.

> Include the `forwardAuthToken` to the destination and your JWT will be forwarded directly as `Authorization` header to the target system.

Reviewer: ```suggestion
Include the `forwardAuthToken` to the destination configuration and your `JWT` will be forwarded directly as `Authorization` header to all the target systems without an extra round trip to the destination service.
```

> All you

Reviewer: Seem like this line is incomplete?

Author: I tried to be more explicit. Have a second look.

Note that the SAP Cloud SDK offers also a `mockTestDestination()` method, which reads in a `systems.json` and `credentials.json` to create destinations.
The advantage of files is that they can be excluded from the repository since they contain sensitive information.

:::tip AuthToken Forwarding

In most cases, the destination service should be used in production to fetch a destination including an authentication token.

> In the case of frequent service to service communication, requesting the destination service every time for a target specific `JWT` would mean a big overhead.

Reviewer: ```suggestion
In the case of frequent service to service communication, requesting the destination service every time for a target-specific `JWT` would mean a big overhead.
```

In such a case you can configure your destination targets to accept the initial request `JWT` and use this for all further calls.

> This makes the call to the destination service for a target specific token redundant.

Reviewer: ```suggestion
This makes the call to the destination service for a target-specific token redundant.
```

> If you configured your target system in such a way, you can use an environment destination to stop the destination lookup right away.

Reviewer: What's `environment destination`?  Should it be set as an environment variable? Can we link to that section?

Reviewer: In the example below, you provide the destination inline still referring to the environment variable.

Author: yes because the methods sets the destination in the envioremnt varialbes with some checks. It is explained above already in the same section.

Include the `forwardAuthToken` to the destination configuration and your initial `JWT` will be forwarded directly as `Authorization` header to the target systems without an extra round trip to the destination service.
You can use the `setTestDestination()` method to set the destination in the environment variables:
```ts
setTestDestination({
  name: 'FORWARD-DESTINATION',
  url: 'https://mys4hana.com',
  forwardAuthToken: true
});

MyReqeust.execute({destinationName: 'myDestination', jwt: 'forwardedJwt'})
```
:::

> Note that this approach is not possible for multi tenant scenarios, because the destination services takes care

Reviewer: ```suggestion
Note that this approach is not possible for multi-tenant scenarios, because the destination services takes care
```

Reviewer: This sentence is still unfinished?
